package.json

```diff
@@ -99,13 +100,17 @@
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
+    "prestart": "node scripts/prebuild.js",
     "start": "DISABLE_ESLINT_PLUGIN=true BROWSER=none GENERATE_SOURCEMAP=false node scripts/start.js",
+    "prebuild": "node scripts/prebuild.js",
```

Comment:
Do you want to add README comments about when a user would need to run the prestart and prebuild / prebuild-windows scripts? There are still some unresolved comments from my last review relating to the README, so you can probably address them all in one commit.

Reply:
Those pre scripts run automatically.

Reply:
Ahh, that's a nice feature, I hadn't realized that. This should be fine then, we'll just need to tell UMMC to run the script to convert the strides CSV file to JSON, which is already documented in the README.